solutions/cses-1132.mdx

```diff
@@ -80,7 +83,98 @@ int main() {
 	}
 	return 0;
 }
+```				 
+</CPPSection>
+	
+<JavaSection>
+	
+<Warning>
+Java exceeds the time limit on two test cases.
+</Warning>
+	
+```java
+import java.util.*;
+import java.io.*;
+
+public class TreeDistancesI {
+	// source -> https://cses.fi/problemset/task/1132/
+
+	static ArrayList<Integer>[] adj;
+	static int [] d1 = new int[200001];  // to store first-max length.
+	static int [] d2 = new int[200001]; // to store second-max length.
+	static int [] c = new int[200001]; // to store child for path of max length.
```

Comment:
85 characters using indents as 8 spaces, 81 characters using indents as 4 spaces. Either way, above 80 character limit.

Reply:
its not a strict limit 81-85 is fine

Reply:
![image](https://user-images.githubusercontent.com/58674441/141366139-d9b49aba-81f1-4657-9bc1-dac22f99392f.png)

Reply:
aadit i don't know if you've been living under a rock but literally everyone else feels that 81-85 is fine- as long as it doesn't wrap on the website it's fine

Reply:
pls not another sans 1 is enough